### PR TITLE
Add Quantum Computing

### PR DESCRIPTION
**Pull request by iremddemir**
## What does this PR do?
Create a subject and add resource(s) for Quantum Computing

### Description
Free Quantum Computing books are added
### Why is this valuable (or not)?
QC is an important area of interest in computer science
### How do we know it's really free?
Published as open access or free book
### For book lists, is it a book? For course lists, is it a course? etc.
yes
## Checklist:
- [+ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md)
- [ +] Search for duplicates.
- [+ ] Include author(s) and platform where appropriate.
- [+ ] Put lists in alphabetical order, correct spacing.
- [+] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-subjects.md [hunk 1]
```diff
@@ -669,6 +670,13 @@ Kerridge (PDF) (email address *requested*, not required)
 * [Type Theory and Functional Programming](https://www.cs.kent.ac.uk/people/staff/sjt/TTFP/)
 
 
+### Quantum Computing
+
+* [Introduction to Classical and Quantum Computing](http://www.thomaswong.net/introduction-to-classical-and-quantum-computing-1e2p.pdf) - Thomas G. Wong (PDF)
+* [Learn Quantum Computation using Qiskit](https://qiskit.org/textbook/preface.html)
+* [Quantum Computing for the Quantum Curious](https://link.springer.com/content/pdf/10.1007%2F978-3-030-61601-4.pdf) - C Hughes,J Isaacson, A Perry, R F. Sun, J Turner (PDF)
```
Comment (davorpa) [inline books/free-programming-books-subjects.md:677]: Fit with CC license

```suggestion
* [Quantum Computing for the Quantum Curious](https://link.springer.com/content/pdf/10.1007%2F978-3-030-61601-4.pdf) - Ciaran Hughes, Joshua Isaacson, Anastasia Perry, Ranbel F. Sun, Jessica Turner (PDF)
```

File: books/free-programming-books-subjects.md [hunk 2]
```diff
@@ -669,6 +670,13 @@ Kerridge (PDF) (email address *requested*, not required)
 * [Type Theory and Functional Programming](https://www.cs.kent.ac.uk/people/staff/sjt/TTFP/)
 
 
+### Quantum Computing
+
+* [Introduction to Classical and Quantum Computing](http://www.thomaswong.net/introduction-to-classical-and-quantum-computing-1e2p.pdf) - Thomas G. Wong (PDF)
+* [Learn Quantum Computation using Qiskit](https://qiskit.org/textbook/preface.html)
```
Comment (davorpa) [inline books/free-programming-books-subjects.md:676]: Add contact name as author. More contributors at https://qiskit.org/textbook/preface.html#Contributors

```suggestion
* [Learn Quantum Computation using Qiskit](https://qiskit.org/textbook/preface.html) - Frank Harkins et al. (HTML)
```

**Review by davorpa — CHANGES_REQUESTED** [review]
Complete some info is required. Apply suggestions as is :wink:

**Comment by iremddemir** [comment]
I have made the changes. Thank you for suggestions!

**Review by davorpa — APPROVED** [review]

**Review by eshellman — CHANGES_REQUESTED** [review]
Excellent additions!

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**